dir_class.py

```diff
@@ -31,19 +32,22 @@ def get_control_string(self):
     def get_music_files_count(self):
         return len(self.music_files)
 
-    def move(self, destination_subdir, destination_dir):
-        for x, y, z in os.walk(current_dir):
+    def move(self, destination_subdir):
+        destination_dir = "zrobione"
+        for x, y, z in os.walk(self.base_dir):
             destination_pth = destination_dir + os.sep + destination_subdir
             os.makedirs(destination_pth, exist_ok=True)
             for f in z:
                 source_pth = x + os.sep + f
                 shutil.move(source_pth, destination_pth)
-            break
```

Comment:
`break` jest potrzebny do przerwania pętli. Rzecz w tym, że `os.walk` działa rekursywnie - to znaczy, że wykonamy tą pętlę katalogu self.base_dir i dla każdego podkatalogu który jest wewnątrz self.base_dir i dla każdego który będzie w którymkolwiek z tych podkatalogów.

Właściwie to mogłabyś przeiterować po self.music_files, ale to może na któreś z kolejnych zadań.